nncf/config/schemata/defaults.py

```diff
@@ -35,6 +35,7 @@
 QUANTIZATION_OVERFLOW_FIX = "enable"
 QUANTIZATION_BITS = 8
 QUANTIZATION_PER_CHANNEL = False
+QUANTIZATION_NARROW_RANGE = False
```

Comment:
Again, another default value for this. As in `nncf/common/quantization/quantizer_propagation/solver.py` and `nncf/common/quantization/quantizer_setup.py`. We need to provide only one place to store these.

Reply:
Default value from the `nncf/config/schemata/defaults.py` is reused in the solver and quantizer setup, please check